### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by matthewdupre**

### CODE REVIEW

File: docs/source/redhat-opens-install.rst [hunk 1]
```diff
@@ -494,6 +496,12 @@ On each compute node, perform the following steps:
     generate the configuration for a single peer connection, which you can
     duplicate and update for each compute host in your mesh.
 
+    To maintain connectivity between VMs if BIRD crashes or is reloaded, enable
+    BIRD graceful restart by adding `-R` to the `ExecStart` line in the BIRD
+    systemd unit file (`/usr/lib/systemd/system/bird.service` and `bird6.service`).
+    Please not that this will not work if BIRD is stopped normally - if you need
```
Comment (alexwlchan) [inline docs/source/redhat-opens-install.rst]: “not” ~> “note”

File: docs/source/ubuntu-opens-install.rst [hunk 2]
```diff
@@ -387,6 +389,12 @@ perform the following steps:
    generate the configuration for a single peer connection, which you can
    duplicate and update for each compute host in your mesh.
 
+   To maintain connectivity between VMs if BIRD crashes or is reloaded, enable
+   BIRD graceful restart by adding `-R` to `BIRD_ARGS` in `/etc/bird/envvars`
+   (you may need to uncomment this option).  Please not that this will not work
```
Comment (alexwlchan) [inline docs/source/ubuntu-opens-install.rst]: “not” ~> “note”

File: docs/source/redhat-opens-install.rst (hunk 1 above)
Comment (eepyaich) [inline docs/source/redhat-opens-install.rst]: @matthewdupre As discussed, I think this text needs to say "will not work if BIRD is restarted" and then you need to provide instructions on how to modify upstart accordingly (which you should also test!).

File: docs/source/ubuntu-opens-install.rst [hunk 3]
```diff
@@ -387,6 +389,12 @@ perform the following steps:
...
+   if BIRD is stopped normally - if you need that function, you should modify
+   the upstart job to kill BIRD on stop.
+
```
Comment (eepyaich) [inline docs/source/ubuntu-opens-install.rst]: Same comment as for Red Hat - this is restart and we need to provide the upstart instructions.

**Comment by eepyaich** [comment]
LGTM

**Pull request merged**